content/_developers/attachments-limitations.md

```diff
@@ -38,38 +42,25 @@ will receive the `HTTP 413 - Request Entity Too Large` error response.
 
 Importing attachments from outside by polling is limited by individual components:
 
--   [SFTP component](/components/sftp/) would poll the files below 100 Megabytes in size, if your file is bigger than that SFTP would not poll it and the platform would not report an error message.
-
--   [AWS S3](/components/aws-s3/) would try to read a file from S3 until its' maximum size is bounded to the available component memory. In case of running into the limit the platform will show an error message `Component run out of memory and terminated`, and if problem persists after a number of restarts the platform will suspend the complete integration flow.
+-   [SFTP component](/components/sftp/) would poll the files below 100 MB in size, if your file is bigger than that SFTP would not poll it and the platform would not report an error message.
```

Comment:
Not really. In case when a file is bigger than 100MB, it throws an error like `File size is 124857600 bytes, it violates the variable MAX_FILE_SIZE, which is currently set to 104857600 bytes `